Plugins/Flow.Launcher.Plugin.Explorer/Search/ResultManager.cs

```diff
@@ -26,7 +28,7 @@ internal static Result CreateFolderResult(string title, string subtitle, string
                 TitleHighlightData = StringMatcher.FuzzySearch(query.Search, title).MatchData,
                 Action = c =>
                 {
-                    if (c.SpecialKeyState.CtrlPressed)
+                    if (c.SpecialKeyState.CtrlPressed || !(Settings.EnabledPathSearchKeyword || Settings.EnableSearchActionKeyword))
```

Comment:
Can you do me a favour and change Settings.EnableSearchActionKeyword all to Settings.EnabledSearchActionKeyword please